pages/team.tsx

```diff
@@ -29,6 +32,18 @@ export default function Team({ members, error }: Props) {
               span: <Text as="span" color="purple.400" />,
             }}
           />
+          {users.map((user, id) => {
```

Comment:
aqui eh so uma sugestao no nome da variavel. o user agora seria somente os membros entao faz sentido mas acho que se ao inves de user fosse member ficaria mais facil pra vc e outras pessoas baterem o olho e identificarem quem eh o user em questao. mas isso eh dboas, so achei relevante sugerir isso mesmo

ainda to lendo tudo no projeto pq to a muito tempo fora dele mas parece que ta ficando show demais mano, bom trabalho! 👍  

edit: isso aqui nao era pra fazer parte do review nao, fiz mer** mesmo haha